repositories.go

```diff
@@ -209,6 +209,7 @@ type Compare struct {
 	Diffs          []*Diff   `json:"diffs"`
 	CompareTimeout bool      `json:"compare_timeout"`
 	CompareSameRef bool      `json:"compare_same_ref"`
+	WebUrl         string    `json:"web_url"`
```

Comment:
Please read: https://go.dev/wiki/CodeReviewComments#initialisms In Go this field should be named `WebURL`